src/components/ChallengeModal/ChallengeModal.tsx

```diff
@@ -143,7 +143,7 @@ export class ChallengeModal extends Modal<ChallengeModalProperties, any> {
             conf: {
                 mode: this.props.mode,
                 username: "",
-                bot_id: data.get("challenge.bot", 0),
```

Comment:
eh, darn inline diff thing doesn't show what was replaced - this went from `data.get("challenge.bot", 0)` to `data.get("challenge.bot")`, ie the default of `0` was dropped and I suspect it's just because you were running through ripping out all of the request `get` calls and did this one by accident.